src/app/view/ChessBoardView.ts

```diff
@@ -4,6 +4,12 @@ import { PiecesBoardPositions } from './Types';
 export interface ChessBoardView extends ViewEventSource {
   showChessBoard(piecesPositions: PiecesBoardPositions): void;
 
+  showAvailableMoves(squaresToHighlight: string[]): void;
```

Comment:
Troche mi sie nie podoba, ze to jest lista stringów. Ale na razie mozemy zostawic... gdyby były na to testy :) ?

Reply:
to ja bym prosił o koło ratunkowe, jakbyśmy to widzieli ? :P

Reply:
jakiś nowy typ tworzymy ?

Reply:
Jak przychodzi np. w SquareWasClicked {x,y} tak samo fajnie chyb abyloby jak juz zachowac to x / y tez na wyjsciu? I gdzies przemapowac na widoku dopiero na idika? Normalnie bym tak nie zostawial, ale ten PR sie przeciaga - mozna to zrobic w kolejnym. Testy an to mamy - to mozna zmergowac.